app-admin/yadm/yadm-2.4.0.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit bash-completion-r1
+
+DESCRIPTION="A dotfile manager for the config files in your home folder"
+HOMEPAGE="https://github.com/TheLocehiliosan/yadm/"
+SRC_URI="https://github.com/TheLocehiliosan/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+DEPEND="
+	test? (
+		  dev-python/pytest
+		  dev-python/pylint
+		  || ( dev-util/shellcheck dev-util/shellcheck-bin )
+		  dev-python/flake8
+		  dev-vcs/git
+	)
+"
+
+RDEPEND="
+	dev-vcs/git
+	app-crypt/gnupg
+"
+
+src_compile() {
+	emake "${PN}.md"
+}
+
+src_test() {
+	# test_encryption: needs write access to /tmp
+	rm -f "test/test_encryption.py"
+	# test_alt, test_unit_template_j2: needs envtpl
+	rm -f "test/test_alt.py"
+	rm -f "test/test_unit_template_j2.py"
+	# test_syntax: needs yamllint (not packaged)
+	rm -f "test/test_syntax.py"
+	# test_compat_alt: known broken, tests deprecated features
+	rm -f "test/test_compat_alt.py"
+
```

Comment:
don't quote literal paths, there are no variables involved.
also `|| die` is missing for each `rm` line.
while -f flag ignores some errors(file not found), it can still return errors if permission is denied or in other situations. try to avoid `rm -f` if not absolutely required

Reply:
Ah, I was wondering what the etiquette was with quotes. Done!